ecosystem/sep-0009.md

```diff
@@ -94,7 +94,7 @@ Name | Type | Description
 `organization.number_of_shareholders` | number | Organization shareholder number
 `organization.shareholder_name` | string | Can be an organization or a person and should be queried recursively up to the ultimate beneficial owners (with KYC information for natural persons such as above)
 `organization.photo_incorporation_doc` | binary | Image of incorporation documents
-`organization.photo_proof_adress` | binary | Image of a utility bill, bank statement with the organization's name and address
+`organization.photo_proof_address` | binary | Image of a utility bill, bank statement with the organization's name and address
```

Comment:
@JakeUrban Any issue changing this at this point?

Reply:
No issues I can think of. LGTM